.node-version

```diff
@@ -0,0 +1 @@
+22
```

Comment:
nits: It seems advisable to pin the minor version as well to ensure compatibility and stability.

```suggestion
22.3.0
```

Reply:
thanks, fixing "minor" version makes sense.

but I'm planning to  free "patch" version to apply security patch automatically.

Reply:
pinned at `22.3`: https://github.com/route06/aws-cost-cli/pull/12/commits/a8143948e3efb25f6455a2353b8a8e92e4eed1b9

Reply:
Sure, that sounds like a good plan.